container-images/tcib/base/os/horizontest/run_horizontest.sh

```diff
@@ -25,6 +25,13 @@ SELENIUM_IMPLICIT_WAIT=30
 [[ -z ${REPO_URL} ]] && REPO_URL="https://review.opendev.org/openstack/horizon"
 [[ -z ${HORIZON_REPO_BRANCH} ]] && HORIZON_REPO_BRANCH="master"
 
+#This function is temporarily added until tempest cleanup is implemented
+function clean_leftover_images {
+    if openstack image list -c Name -f value --os-cloud default ; then
```

Comment:
I'm just thinking. Is the if-statement needed?

Reply:
Not a need here . let me remove it